bfh/transformations.py

```diff
@@ -111,18 +111,19 @@ def __init__(self, *args, **kwargs):
         self.path = args
         self.kwargs = kwargs
         self.required = kwargs.get('required', False)
+        self.default = kwargs.get('default')
 
     def __call__(self, source):
         return self.function(source)
 
     def _get_from_dict(self, source, path):
         if not self.required:
-            return source.get(path)
+            return source.get(path) or self.default
         return source[path]
 
     def _get_from_obj(self, source, path):
         if not self.required:
-            return getattr(source, path, None)
+            return getattr(source, path, None) or self.default
```

Comment:
I initially had it that way, but I can't - there is an issue with nested paths, e.g. `Get("outer", "middle", "inner")`.